src/Avalonia.Controls/Documents/Bold.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Avalonia.Media;
+
+namespace Avalonia.Controls.Documents
+{
+    public sealed class Bold : Span
```

Comment:
we might need some xml comment for this?

Reply:
Y will add them. The code isn't polished yet.